packages/workbox-routing/NavigationRoute.mjs

```diff
@@ -18,36 +18,35 @@ import {Route} from './Route.mjs';
 import './_version.mjs';
 
 /**
- * NavigationRoute is a helper class to create a [Route]{@link
- * module:workbox-routing.Route} that matches for browser navigation requests,
- * i.e. requests for HTML pages.
+ * NavigationRoute makes it easy to create a [Route]{@link
+ * module:workbox-routing.Route} that matches for browser navigation requests.
```

Comment:
Can you link to https://developers.google.com/web/fundamentals/primers/service-workers/high-performance-loading#first_what_are_navigation_requests?

(One of the reasons I went into detail in that article is because I hadn't seen another thorough explanation of what a navigation request is and isn't.)